packages/dataviews/src/types.ts

```diff
@@ -153,6 +154,14 @@ export type Field< Item > = {
 	 */
 	filterBy?: FilterByConfig | undefined;
 
+	/**
+	 * Validation schema for the field.
+	 */
+	validationSchema?: {
```

Comment:
Do we need this top level thing? It seems this is moving us away from JSON schema "format" where the field name, id, type... are mixed with the specific constraints like "minLength"... so maybe we can just move the constraints top level (but make them dependent on the field type for most of them)

Reply:
I don’t have a strong preference on this 👍. However, there may be some options that don’t belong in the JSON schema, such as the[ onTouched property](https://github.com/WordPress/gutenberg/pull/67357#discussion_r1871099612).

Reply:
Agree to both things (make min/max top-level) plus remove onTouched for now (if we ever need something like that, we can create a top-level validation config to group these validation-specific things, like we have in `filterBy`).